OpenLiberty/guide-cors
# Dev mode

**Required changes:**
- use the ideal(simplified) pom.xml
- update the Readme content to use dev mode
  - keep the flow same 
- convert `*Test` class to `*IT` 
- convert to use JUnit 5
  - use JUnit5 test order if need
- make sure to correctly configure the ports in test classes, server.xml, and pom.xml
- update travis test
   - `mvn install` cannot be used to verify anymore

Also added SEO title and description.
